### DISCUSSION TITLE
Combining the ‘targets’ package within Quarto.

### DISCUSSION THREAD

**Discussion by JohnStaples** [quarto-dev/quarto-cli#4325] [Q&A]
Any known issues. Any body had a good outcome of which is a complex code development?
With anticipatory thanks …

**Comment by mcanouil** [comment]
Not clear what you are really asking here.
`targets` supports Quarto which is described in <https://books.ropensci.org/targets/literate-programming.html>.

**Comment by JohnStaples** [comment]
Hi mcanouil
I was looking for confirmation from developers of Quarto apps that included ‘targets’ if they successfully created a complex targeted design. I have read the ‘Literate Programme’ chapter, along with all the additional info I could find. I noted the advice to start ‘small’, and expand from there.
This I intend to do …

**Reply by mcanouil** [reply]
I am pretty sure Quarto developers don't try every R packages.
You are more likely to find `targets` related things in `targets`/`tarchetypes` repositories.

This being said, Quarto CLI not not `RMarkdown` since the former ran outside R and can execute R code while the latter is ran within R, thus don't expect to be able to do exactly the same.

But again, in case of issues/questions about `targets`, it is in my belief that you should go to `targets`/`tarchetypes` repositories, especially because Will Landau is really active to address questions/issues regarding his tools.
